lib/resources/r4/related_person_patch.yaml

```diff
@@ -131,6 +142,44 @@ operations:
         <li>Requires a test operation to be provided for the `Relationship.id` field of the relationship to be removed.</li>
       </ul>
 
+  - name: replace-relationship-extensions
+    path: /relationship/{index}/extension
+    operation: replace
+    type: string
+    url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson.relationship
+    description: Replace the `period` and `relation` of the relationship at the given <code>{index}</code>.
+    example: |
+      {
+        "op": "replace",
+        "path": "/relationship/0/extension",
+        "value": [
+          {
+            "valuePeriod": {
+              "start": "2016-12-30T22:26:08.000Z",
+              "end": "2036-12-30T22:26:08.000Z"
+            },
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/period"
+          },
+          {
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40",
+                  "code": "155",
+                }
+              ]
+            },
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/relation"
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>Requires a test operation to be provided for the `Relationship.id` of the relationship whose `period` and `relation` is intended to be replaced.</li>
+        <li>The `start` and `end` when provided must include a time component with a timezone.</li>
+        <li>When either `start` or `end` are not provided its value will be unset.</li>
+      </ul>
```

Comment:
Should we also not mention in the notes section about the relation extension, something like:
```
The relationship.relation extension must contain exactly one proprietary system.
The relationship.relation extension cannot contain a standard system. (assuming the standard code support code is not yet added)
```
I think we can get a clarification on that.

Reply:
Added https://github.com/cerner/fhir.cerner.com/pull/662/commits/f940696cabd6f27c86a9e0b4b1e62676f834e0ed#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR179-R180

Reply:
@alexbeckner1 , need your +1, if this change is required.

Reply:
That's fine to add that here, but I left them out because I figured we could just add notes when we did the fhir.cerner.com for the standard code support.

Reply:
@AnanyaRamesh14  I unresolved the comment, since we are now doing the standard code support in the same PR,
should we now add the note to say:

`The relationship.relation extension must contain exactly one proprietary system or a standard system.
`
or @alexbeckner1  if you have any suggestions on the wordings.

Reply:
Whatever gets landed on in the above comment can go here too.

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/662/commits/062a7f28bf665d671d55b5d50c1f8fa7067a2349#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR181-R182

Reply:
Doesn't look like the changes have been done?

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/662/commits/efb7ac07367d7009d3785e8e9562c73fbb6f33b5#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR183